crt_portal/static/js/complaint_view_filters.js

```diff
@@ -73,6 +96,14 @@
 
       return memo;
     }, []);
+    if (resetGroupParams) {
+      for (let i = 0; i < newParams.length; i++) {
+        if (newParams[i].includes('group_params')) {
+          newParams.splice(i, 1);
+        }
+      }
+    }
+    return newParams;
```

Comment:
This can be done with a filter - any time you're doing "splice", "remove", or "filter" on a list, you can usually rewrite it as a `Array.filter`

```suggestion
    if (!resetGroupParams) return newParams;
    return newParams.filter(param => !param.includes('group_params'));
```